### ISSUE TITLE
Professor Putricide hc

### ISSUE THREAD

**Issue by alexuktc** [dalaranwow/dalaran-wow#5581]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
It goes like this. You go into transition phase and both the oozes spawn and assign targets. You do not kill the orange ooze fast enough and it retargets. If you are particularly unlucky it will retarget the tank who is on the boss and has aggro on him. Usually that is fine, he can kite it like a normal player would. 

The problem on this server is that if the tank is chased by the orange ooze and PP goes out of phase and attacks him the orange ooze will release its stacks on the raid ragardless of the range between the tank and the ooze. 

TL:DR:
If PP hits the tank while he is chased by the orange ooze the orange ooze will consider it as if it has hit him regardless of range.

**Expected behaviour**: Tell us what should happen instead
This should not happen. I am not even sure if the orange ooze should be able to target the player holding aggro on the boss in the first place. 

**Steps to reproduce the problem**:

1. Have the orange ooze on the aggro holder at the end of the transition phase
2.  Have PP hit the tank while the ooze is away.
3. 

**Include proofs for this behaviour**
Sadly we did not anticipate such a thing so we were not actively looking for it. It is possible that we had a streamer in the raid. If that is the case I will have him provide us with the video of it happening.

**Comment by Kookaburra129** [comment]
Can confirm, was in this raid. Caught us completely by surprise.

**Comment by LoveDalaranWow** [comment]
Can confirm too, happened last week in our raid
